### PR TITLE
Feature/light hero

### PR DESCRIPTION
**Pull request by federivero**
Complete component (desktop, mobile)

### CODE REVIEW

File: components/modules/LightHero/LightHero.module.scss [hunk 1]
```diff
@@ -1,7 +1,18 @@
...
+  @apply container default-grid bg-cream;
+
+  .headline{
+    @apply typo-h2 text-blue col-start-1 col-span-2 mt-50;
+    @apply md:col-start-1 md:col-span-8 md:mt-120;
+  }
+
+  .subline{
+    @apply text-black col-start-1 col-span-2 text-20 mb-25;
+    @apply md:col-start-1 md:col-span-4 md:mb-120;
```
Comment (alexscro) [inline components/modules/LightHero/LightHero.module.scss:13]: @federivero 

we need to move spacing downwards always to the container or outer parent div.
There might be a case that the subline won't be in the CMS and only the headline is present which would then result in no spacing down in that case.

File: components/modules/LightHero/LightHero.tsx [hunk 2]
```diff
@@ -1,12 +1,18 @@
...
 const LightHeroModule: FunctionComponent<{ module: ILightHero }> = ({
   module,
 }) => {
-  console.log(module)
-  return <div className={`${styles.root} container`}>Lighthero Module</div>
+  const { headline, subline } = module
+  return (
+    <div className={`${styles.root} container`}>
+      <div className={c(styles.headline)}>{headline}</div>
+      <div className={c(styles.subline)}>{subline}</div>
```
Comment (alexscro) [inline components/modules/LightHero/LightHero.tsx:13]: please validate the prop: @federivero 

{headline && ....}
{ subline && <div...}

File: components/modules/LightHero/LightHero.module.scss (hunk 1 above)
Comment (federivero) [reply components/modules/LightHero/LightHero.module.scss:13]: Got it, my bad, I wasn't even validating the properties

File: components/modules/LightHero/LightHero.tsx (hunk 2 above)
Comment (federivero) [reply components/modules/LightHero/LightHero.tsx]: Done, thank you for pointing that out.

**Pull request merged**